developer-tools.md

```diff
@@ -435,6 +435,46 @@ Error:(147, 9) value q is not a member of StringContext
         q"""
         ^ 
 ```
+<h4>Remote Debug Unit Test</h4>
+This part will show you how to debug unit test in SBT with the help of remote debug functionality which provided by IntelliJ.
+
+<h5>Set up Remote Debug Configuration</h5>
+Follow <i>Run > Edit Configurations > + > Remote</i> to open a default Remote Configuration template:
+<img src="/images/intellij_remote_debug_configuration.png" style="width: 75%; max-width: 660px;" />
+
+Normally, the default values should be good enough to use. Make sure that you choose <b>Listen to remote JVM</b>
+as <i>Debugger mode</i> and select the right <i>Command line arguments for remote JVM</i> as per your JDK version.
```

Comment:
`select the right JDK version to generate proper <i>Command line arguments for remote JVM</i>`